portfolio/src/main/webapp/comment.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
You don't need this file since all of your commenting logic is handled via `index.html` and `script.js`